operators/src/adapters/sparse_tiles_fill_adapter.rs

```diff
@@ -340,8 +340,9 @@ where
                         }
                         // 4. The received tile has a TimeInterval that starts after the current TimeInterval and is not directly connected to the current TimeInterval.
                         else {
+                            debug_assert!(this.sc.current_time.end() <= tile.time.start());
```

Comment:
what if the tile is the first in the grid run and there is actually a gap after the previous one? Then this would not hold. Actually the comment in 4 states exactly this.

Reply:
yes it does. `sc` is the state container so the time in the container aka the latest time so far must always be smaller then the time of the newly arrived `tile`.
"4." states that "The received tile has a TimeInterval that starts after the current TimeInterval" so it must hold.

Reply:
yes it does. sc is the state container so the time in the container aka the latest time so far must always be smaller then the time of the newly arrived tile.
"4." states that "The received tile has a TimeInterval that starts after the current TimeInterval" so it must hold.